### ISSUE TITLE
Improve accessibility of keywords input in adminDataset create form

### ISSUE THREAD

**Issue by luistoptal** [gigascience/gigadb-website#1573]
## User story

http://gigadb.gigasciencejournal.com:9170/adminDataset/create

![image](https://github.com/gigascience/gigadb-website/assets/143437854/6774a262-4c6f-4a47-8445-51333eb183a2)

The input presents some issues:

* On chrome and edge, keyboard navigation does not work properly: filling some keywords, tabbing to next input and then back again, does not restore focus to the keywords input
* On focus, screen reader does not announce the label
* Since this is a non-standard input, a description might be useful, particualrly for screen readers

## Additional info

related ticket #1467
